view/frontend/web/js/model/spi.js

```diff
@@ -121,7 +125,57 @@ define([
                             return {card: scaResult};
                         }
                         throw new Error('Unsupported payment type');
-                    }.bind(this)
+                    }.bind(this),
+                    'onRequireOrderData' : async function (requirements) {
+                        const payload = {};
+
+                        for (const rq of requirements) {
```

Comment:
Nitpick: short variable names should not be used. For clarity, `rq` should be `requirement`.

Reply:
Updated to `requirement`